codice/alliance
# CAL-154 Automates the Sample NSILI Client tests cases into the itests

#### What does this PR do?

This PR automates the Sample NSILI Client tests cases into the itests to expand coverage.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@lcrosenbu @bdeining @troymohl @glenhein @mweser @codice/test 

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@kcwire
@lessarderic

#### How should this be tested?

1. Confirm itests pass on full build.
2. Confirm that the Sample NSILI Client tests cases are still available to be run from the command line (see [README](https://github.com/codice/alliance/blob/master/distribution/sdk/sample-nsili-client/README.md)). The functionality of the Sample NSILI Client tests cases should be the same as before. NOTE: The NSILI App must be installed for the NSILI Endpoint to be active.

#### Any background context you want to provide?

The goal of the Sample NSILI Client is to test that a client can make queries and orders to the NSILI Endpoint.

#### What are the relevant tickets?

[CAL-154](https://codice.atlassian.net/browse/CAL-154)

#### Screenshots (if appropriate)
#### Checklist:
